### PR TITLE
Rename CLIConfig, CLIParams

### PR DESCRIPTION
**Pull request by SoManyHs**

### CODE REVIEW

**Comment by SoManyHs** [comment]
`make build && make test` pass

**Comment by SoManyHs** [comment]
(Note: For some reason, #456 got closed in the rebase so re-opening as a new PR)

File: ecs-cli/modules/cli/cluster/cluster_app.go [hunk 1]
```diff
@@ -192,20 +192,25 @@ func createCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfnClien
 
 	}
 
+	// Check if cluster is specified
+	if commandConfig.Cluster == "" {
+		return fmt.Errorf("Please configure a cluster using the configure command or the '--%s' flag", flags.ClusterFlag)
```
Comment (allisaurus) [inline ecs-cli/modules/cli/cluster/cluster_app.go]: w/o this check, what happens if a cluster isn't specified? does it always error out later, or are there circumstances when cluster's not required (like pushing ECR images) ?

File: ecs-cli/modules/cli/cluster/cluster_app.go [hunk 2]
```diff
@@ -307,23 +312,27 @@ func createEmptyCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfn
 		logrus.Warn("Force flag is unsupported when creating an empty cluster.")
 	}
 
+	if commandConfig.Cluster == "" {
+		return fmt.Errorf("Please configure a cluster using the configure command or the '--%s' flag", flags.ClusterFlag)
```
Comment (allisaurus) [inline ecs-cli/modules/cli/cluster/cluster_app.go]: same comment as on line 197

**Pull request merged**